src/main/java/orm/QueryRenderer.java

```diff
@@ -0,0 +1,61 @@
+package orm;
+
+import orm.dsl.condition.Condition;
+import orm.util.CollectionUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class QueryRenderer {
+
+    /**
+     * 컬럼들을 콤마로 구분하여 반환
+     */
+    public String singleColumnListDotted(List<? extends TableField> columns) {
+        return columns.stream()
+                .map(TableField::getFieldName)
+                .collect(Collectors.joining(","));
+    }
+
+    /**
+     * 컬럼들을 콤마로 구분하여 반환, 양 끝을 ()로 감싼다.
+     */
+    public String singleColumnListEnclosed(List<? extends TableField> columns) {
+        return "(%s)".formatted(columns.stream()
+                .map(TableField::getFieldName)
+                .collect(Collectors.joining(",")));
+    }
+
+    /**
+     * bulk insert 를 위해 (...), (...), (...) 형태로 변환
+     */
```

Comment:
저는 처음에 무슨 의미인지 잘 알지 못했어요. 그래서 코드를 다시 보아야했어요.
동민님은 주석을 어떨때 보통 사용하시나요?

저는 아래와 같이 표현할것 같아요.
```suggestion
    /**
     * 대량 삽입 SQL 문에 대한 값의 문자열 표현을 렌더링 합니다.
     *
     * <p>예를 들어, 이러한 형태가 됩니다 : (id,name,age) </p>
     **/
```

Reply:
public 함수에 대해서는 그 함수만의 개별 테스트를 만드는 게 좋을것 같아요. 
예를 들어, (id,name,age)를 출력하는 기능에서 문제가 생겼다고 해서 전체 테스트를 다 돌리는 건 비효율적일 수 있어요. 전체 테스트는 검토해야 할 부분이 너무 많아서 실제 문제가 어디에 있는지 찾기 어려울 수 있지않을까요? 대신 해당 기능에 대한 단위 테스트가 있다면, 그것만 실행해서 빠르게 문제를 확인하고 해결할 수 있을것 같아요 🤔

Reply:
넵 이부분은 다른 상위 테스트로에서도 커버할 수 있다고 생각했었는데요.
말씀하신대로 테스트코드가 있으면 좋겠네요 :)